flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/common/CommonExecSink.java

```diff
@@ -351,6 +356,21 @@ private int deriveSinkParallelism(
                 sinkParallelism);
     }
 
+    private Transformation<RowData> applyTimestampTransformation(
```

Comment:
nit: call this `applyRowtimeTransformation` because "timestamp" sounds very generic given all the different types of timestamps we have in the planner